api/v1/views/places_reviews.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.place import Place
+from models.review import Review
+
+
+@app_views.route('/places/<place_id>/reviews',
+                 methods=['GET'], strict_slashes=False)
+def get_places_reviews_id(place_id):
+    """ GET method to list every review of a place """
+    catch_place_review = storage.get('Place', place_id)
+    if catch_place_review is None:
+        abort(404)
+    place_review_list = []
+    for place_review in catch_place_review.reviews:
+        place_review_list.append(place_review.to_dict())
+    return jsonify(place_review_list)
+
+
+@app_views.route('/reviews/<review_id>', methods=['GET'], strict_slashes=False)
+def get_reviews_id(review_id):
+    """ GET method to list every review """
+    catch_review = storage.get('Review', review_id)
+    if catch_review is None:
+        abort(404)
+    return jsonify(catch_review.to_dict())
+
+
+@app_views.route('/reviews/<review_id>',
+                 methods=['DELETE'], strict_slashes=False)
+def del_review_id(review_id):
+    """ DELETE method to review a review by id """
+    catch_review = storage.get('Review', review_id)
+    if catch_review is None:
+        abort(404)
+    storage.delete(catch_review)
+    storage.save()
+    return jsonify({}), 200
+
+
+@app_views.route('/places/<place_id>/reviews',
+                 methods=['POST'], strict_slashes=False)
+def post_reviews(place_id):
```

Comment:
saw in other scripts that this definition could use a better name. Something like "post_places_id_reviews"

Reply:
Change fixed